qemu/tests/cfg/check_block_size.cfg

```diff
@@ -11,12 +11,19 @@
     index_enable = no
     image_name_stg = "images/check_block_size_image"
     image_size_stg = 20G
+    image_verify_bootable = no
     force_create_image_stg = yes
     drive_serial_stg = "TARGET_DISK0"
     cdroms += " unattended"
     unattended_delivery_method = cdrom
     chk_phy_blk_cmd = "cat /sys/block/%s/queue/physical_block_size"
     chk_log_blk_cmd = "cat /sys/block/%s/queue/logical_block_size"
+    chk_blks_cmd_windows = "powershell "get-disk|format-list""
+    variants:
+        - extra_cdrom_ks:
+            no 4096_4096
+        - base:
+            only 4096_4096
```

Comment:
For base, there is no installation.
03:07:38 ERROR| TestError: Could not find the specifiedvirtio block device.    [context: (run)]

03:07:38 ERROR| ERROR 1-smp_8.16384m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.Win2016.x86_64.io-github-autotest-qemu.check_block_size.4096_4096.base -> TestError: Could not find the specifiedvirtio block device.    [context: (run)]
03:07:38 INFO |